dashboard/app/views/layouts/_header.html.haml

```diff
@@ -30,7 +30,10 @@
         .row.header_row
           .header_left
             .header_logo
-              = link_to(image_tag('codeorg-studio-logo.png'), '/')
+              - if current_page?(root_url)
+                = link_to(image_tag('logo.png'), CDO.code_org_url('/'))
+              - else
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace